internal/schemautil/userconfig/userconfig_test.go

```diff
@@ -11,16 +12,15 @@ import (
 	"github.com/ettle/strcase"
 	"golang.org/x/exp/maps"
 	"golang.org/x/exp/slices"
-	"gopkg.in/yaml.v3"
 )
 
 // generateSchema is a function that generates Terraform schema via its map representation.
 func generateSchema(n string, m map[string]interface{}) {
 	np := fmt.Sprintf("%ss", n)
 
-	f := jen.NewFile("user_config")
+	f := jen.NewFile("dist")
```

Comment:
Let's move it to a param, on :76 too.

Reply:
sorry, but why do we need this?

Reply:
Because it's a hardcoded config. If you try to write a test for this, you will face the problem that you can't use tmp or other dir for that.

Reply:
well then we will make it a param when this is needed, overengineering is generally a bad idea